test/dump/call_ref.txt

```diff
@@ -0,0 +1,108 @@
+;;; TOOL: run-objdump
+;;; ARGS0: -v --enable-function-references
+
+(module
+  (func (export "main") (result i32)
+    i32.const 10
+    ref.func $foo
```

Comment:
What would it look like to receive a `ref.func` as an argument?   Or is that coming in the next phase/PR?

Reply:
Some new tokens need to be introduced to support high order functions so let's move that in the next PR